apis/infrastructure/v1beta1/webhook_suite_test.go

```diff
@@ -34,6 +34,11 @@ import (
 // These tests use Ginkgo (BDD-style Go testing framework). Refer to
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
+const (
+	bundleLookupTag        = "v0.1.0_alpha.2"
```

Comment:
@dharmjit Do we need `bundleLookupTag` anymore? If not, can we create an issue to remove this field?

Reply:
Just an add-on, it's used 4 times in current code

Reply:
Yes it's not required anymore. Created this Issue - https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/issues/567

Reply:
Thanks, then I mark this conversation resolved